### PR TITLE
Cleanup sf

### PR DESCRIPTION
**Pull request by reid-andrew**
### Type of change made:
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling - no new features
### Why is this change required? What problem does it solve?
- Refactor to meet standards and format .
- Correct bug with posting to charity_image_score
### Were there any challenges that arose while implementing this feature? If so, how were they addressed?
- n/a

### CODE REVIEW

File: application/controllers/charities.py [hunk 1]
```diff
@@ -48,9 +47,10 @@
     location=['json']
 )
 
+
 class CharityResources(Resource):
     def get(self, search_term=None):
-        search_term  = "" if search_term == None else search_term
+        search_term = "" if search_term is None else search_term
```
Comment (kmcgrevey) [inline application/controllers/charities.py:53]: beefier Python syntax!

File: tests/test_items.py [hunk 2]
```diff
@@ -101,7 +100,6 @@ def test_create_items(self):
                 'charity': 'Big Cat Rescue',
                 'charity_url': "http://www.thisisatotallyligiturl.com",
                 'charity_score': 4,
-                'charity_score_image': "img.url",
                 'charity_score_image': "https://d20umu42aunjpx.cloudfront.net/_gfx_/icons/stars/4stars.png",
```
Comment (kmcgrevey) [inline tests/test_items.py:103]: oh -- actual image url

File: application/controllers/items.py [hunk 3]
```diff
@@ -181,15 +179,14 @@ def put(self, item_id=None):
             if 'charity_score' in request.json:
                 item.charity_score = request.json['charity_score']
             if 'charity_score_image' in request.json:
-                item.charity_score = request.json['charity_score_image']
+                item.charity_score_image = request.json['charity_score_image']
```
Comment (reid-andrew) [inline application/controllers/items.py:182]: This is the correction I made - looks like we made a copy/paste mistake earlier this morning.

Comment (kmcgrevey) [reply application/controllers/items.py:182]: oh man -- good catch

**Review by kmcgrevey — APPROVED** [review]

**Pull request merged**